package.json

```diff
@@ -39,7 +41,42 @@
     "@types/react": "16.9.53"
   },
   "eslintConfig": {
-    "extends": "@redwoodjs/eslint-config"
+    "extends": "@redwoodjs/eslint-config",
+    "rules": {
+      "import/order": [
+        "error",
```

Comment:
Could we make this a warning instead, so that editors should still auto-format it for people that have the correct extensions installed, but it won't block us from merging a PR if it's not 100% adherence to the standard.